src/main/java/org/meme/corp/client/Client.java

```diff
@@ -0,0 +1,30 @@
+package org.meme.corp.client;
+
+import java.util.List;
+import java.util.Map;
+import org.meme.corp.dto.EventDto;
+import org.meme.corp.dto.TransactionDto;
+import org.meme.corp.dto.UserDto;
+
+public interface Client {
+
+  //================Events===================
+  EventDto createNewEvent(String name, List<Long> participantIdList);
+
+  List<EventDto> listEvents(Long chatId);
```

Comment:
I suppose we also need client name for chat because our Chat table has the composite key.